docs/references/architecture/adr-118-mempool-lanes.md

```diff
@@ -462,24 +462,38 @@ configurations are:
 - `MaxTxsBytes`, the maximum total number of bytes of the mempool, and
 - `MaxTxBytes`, the maximum size in bytes of a single transaction accepted into the mempool.
 
+However, we still need to enforce limits on each lane's capacity. Without such limits, a
+low-priority lane could end up occupying all the mempool space. Since we want to avoid introducing
+new configuration options unless absolutely necessary, we propose two simple approaches for
+partitioning the mempool space.
+
+1. Proportionally to lane priorities: This approach could lead to underutilization of the mempool if
+   there are significant discrepancies between priority values, as it would allocate space unevenly.
+2. Evenly across all lanes: Assuming high-priority transactions are smaller in size than
+   low-priority transactions, this approach would still allow for more high-priority transactions to
+   fit in the mempool compared to lower-priority ones.
+
+For the MVP we have chosen the second approach. Note that the maximum lane capacities apply both to
+the number of transactions and the size in bytes. 
```

Comment:
Should we mention here the option the user has to increase the mempool capacity if they see that the assigned slot is too small ? Sergio mentioned that during the meeting where we discussed this.

Reply:
Not sure whether this should go into the ADR or in a potential user guide but the idea that for a first attempt to mitigate the big transactions attack, users could have two lanes of the same priority but assign big txs only to one of those so they do not fill up the mempool.

Reply:
I added your first suggestion to the text. The second one I would leave it for the user guide.